tests/unit/test_srp_server.cpp

```diff
@@ -533,7 +533,7 @@ void TestSrpServerReject(void)
     srpClient->EnableAutoStartMode(nullptr, nullptr);
     VerifyOrQuit(srpClient->IsAutoStartModeEnabled());
 
-    AdvanceTime(2000);
+    AdvanceTime(15 * 1000);
```

Comment:
Should we define a constant for this?

Reply:
We can, but I'm not sure if it will be very useful. It's only going to be used in this specific unit test, and we use numbers everywhere else when calling `AdvanceTime()` (which I actually prefer as I find it easier to read and reason about each test step).